docs/excel/excel-add-ins-pivottables.md

```diff
@@ -101,54 +102,142 @@ Adding the **Farm** column pivots all the sales around each farm. Adding the **T
+
+### Create a slicer
+
+Slicers are created by adding them to `slicers` property (which is of type [SlicerCollection](/javascript/api/excel/excel.slicercollection)) of a `Workbook` or `Worksheet` object. Slicers are added with the `SlicerCollection.add` method. This has three parameters:
+
+- `slicerSource`: The data source on which the new slicer is based. It can be a `PivotTable`, `Table`, or string representing the name or ID of a `PivotTable` or `Table`.
+- `sourceField`: The field in the data source by which to filter. It can be a `PivotField`, `TableColumn`, or string representing the name or ID of a `PivotField` or `TableColumn`.
+- `slicerDestination`: The worksheet where the new slicer will be created. It can be a `Worksheet` object or the name or ID of a `Worksheet`. This parameter is unnecessary when the `SlicerCollection` is accessed through `Worksheet.slicers`. In this case, the collection's worksheet is used as the destination.
+
+The following code sample shows a new slicer added to the **Pivot** worksheet. The slicer's source is the **Farm Sales** PivotTable and filters using the **Type** data. The slicer is also named **Fruit Slicer** for future reference.
+
+```js
+Excel.run(async (context) => {
+    var sheet = context.workbook.worksheets.getItem("Pivot");
+    var slicer = sheet.slicers.add(
+        "Farm Sales" /* The slicer data source. For PivotTables, this can be the PivotTable object reference or name. */,
+        "Type" /* The field in the data to filter by. For PivotTables, this can be a PivotField object reference or ID. */
+    );
+    slicer.name = "Fruit Slicer";
+    return context.sync();
+});
+```
+
+### Filter items with a slicer
+
+The slicer filters the PivotTable with items from the `sourceField`. The `Slicer.selectItems` method sets the items that remain in the slicer. These items are passed to the method as a `string[]`, representing the items' keys. Any rows containing those items remain in the PivotTable's aggregation. Subsequent calls to `selectItems` set the list to those calls' keys.
+
+> [!NOTE]
+> If `Slicer.selectItems` is passed an item that's not in the data source, an `InvalidArgument` error is thrown. The contents can be verified through the `Slicer.slicerItems` property, which is a [SlicerItemCollection](/javascript/api/excel/excel.sliceritemcollection).
+
+The following code sample shows three items being selected for the slicer: **Lemon**, **Lime**, and **Orange**.
+
+```js
+Excel.run(async (context) => {
+    var slicer = context.workbook.slicers.getItem("Fruit Slicer");
+    // Anything other than the following three values will be filtered out of the PivotTable for display and aggregation.
+    slicer.selectItems(["Lemon", "Lime", "Orange"]);
+    return context.sync();
+});
+```
+
+To remove filters from the slicer, use the `Slicer.clearFilters` method, as shown in the following sample. Note that this removes all the filters.
```

Comment:
Suggest opening with "To remove all filters from the slicer,..."  and then removing the sentence 'Note that this removes all the filters." (which is now unnecessary, since you specified "To remove **all** filters...").